_emails/newsletter-2021-05.html

```diff
@@ -0,0 +1,61 @@
+---
+title: Origami Newsletter, May 2021
+companion_post_url: https://origami.ft.com/blog/2021/06/01/newsletter/
+---
+
+<!-- TL;DR -->
+{% capture text %}
+The Origami specification is no more
+{% endcapture %}
+
+{% include email/tldr.html content=text %}
+
+{% include email/h2.html content="The Origami specification is no more" %}
+{% capture text %}
+The team spent most of the month revisiting the Origami Specification. We were trying to separate the parts of it that were developer advice from the parts that were relied on by our tools and services. After reducing the specification from 35 pages to 3, we had one last meeting in which we ultimately decided to retire the specification entirely.
```

Comment:
Another little reminder of the npm work? Connect it to why we were looking at the spec now
```suggestion
As part of the [Bower to npm migration](/blog/2021/01/18/deprecating-bower-and-origami-via-npm/), the team spent most of the month revisiting the Origami Specification. We were trying to separate the parts of it that were developer advice from the parts that were relied on by our tools and services. After reducing the specification from 35 pages to 3, we had one last meeting in which we ultimately decided to retire the specification entirely.
```